appium/options/ios/xcuitest/wda/wda_options.py

```diff
@@ -0,0 +1,1071 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from datetime import timedelta
+from typing import Dict, List, Optional, Union
+
+from appium.options.base_options_descriptor import OptionsDescriptor
+from appium.options.common.supports_capabilities import SupportsCapabilities
+
+
+class WDATransformTimeout:
+    @staticmethod
+    def transform_wda_timeout_get(value: Optional[int]) -> Optional[timedelta]:
+        return None if value is None else timedelta(seconds=value)
+
+    @staticmethod
+    def transform_wda_timeout_set(value: Union[timedelta, int]) -> Union[int, float]:
```

Comment:
should be the value of type float as well?

Some WDA caps are different because they accept float seconds instead of milliseconds and there should be a way to set float values there and get timedeltas in response

Reply:
also any int number is a valid float  value